Source/WebCore/dom/messageports/MessagePortChannelRegistry.cpp

```diff
@@ -51,14 +51,10 @@ void MessagePortChannelRegistry::messagePortChannelCreated(MessagePortChannel& c
 {
     ASSERT(isMainThread());
 
-    auto result = m_openChannels.ensure(channel.port1(), [channel = &channel] {
-        return channel;
-    });
+    auto result = m_openChannels.add(channel.port1(), channel);
     ASSERT(result.isNewEntry);
 
-    result = m_openChannels.ensure(channel.port2(), [channel = &channel] {
-        return channel;
-    });
+    result = m_openChannels.add(channel.port2(), channel);
     ASSERT(result.isNewEntry);
```

Comment:
Don’t we need to do `ASSERT_UNUSED`?

Reply:
I was also a little surprised when I modified this code but the compiler apparently doesn’t complain. I can switch to assert unused though to be safe.